homeassistant/components/google_assistant_sdk/notify.py

```diff
@@ -24,13 +24,13 @@
 }
 
 
-def broadcast_commands(language_code: str):
+def broadcast_commands(language_code: str) -> tuple[str, str]:
     """Get the commands for broadcasting a message for the given language code.
 
     Return type is a tuple where [0] is for broadcasting to your entire home,
     while [1] is for broadcasting to a specific target.
     """
-    return LANG_TO_BROADCAST_COMMAND.get(language_code.split("-", maxsplit=1)[0])
+    return LANG_TO_BROADCAST_COMMAND[language_code.split("-", maxsplit=1)[0]]
```

Comment:
This would now fail if a language code isn't present in the mapping dict. However, this change just moves the error as the caller already expects this to always be there.
https://github.com/home-assistant/core/blob/bb03579bd9aff6ab213ea4c75e3f1fd4db48c466/homeassistant/components/google_assistant_sdk/notify.py#L65-L71

It's even a slight improvement as it will be a `KeyError` now instead of the `TypeError`.

Reply:
This should never happen. I even have a test to ensure all languages have a mapping see https://github.com/home-assistant/core/blob/2b43271c3bf9f9906231c9e9d42aaaae0b072fc0/tests/components/google_assistant_sdk/test_notify.py#L150-L163